crates/proof-of-sql/sol_src/base/LagrangeBasisEvaluation.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+contract LagrangeBasisEvaluation {
+    function computeTruncatedLagrangeBasisSum(uint256 length0, bytes memory point0, uint256 numVars0, uint256 modulus0)
+        public
+        pure
+        returns (uint256 result0)
+    {
+        // solhint-disable-next-line no-inline-assembly
```

Comment:
perhaps you want to add this to your solhint:   `"no-inline-assembly": "off",` to avoid repeatedly using // solhint-disable-next-line rule